R/rdeephaven/src/client.cpp

```diff
@@ -256,7 +257,7 @@ class TableHandleWrapper {
             abs_sort = std::vector<bool>(cols.size(), abs_sort[0]);
         }
 
-        for(int i = 0; i < cols.size(); i++) {
+        for(std::size_t i = 0; i < cols.size(); i++) {
```

Comment:
Is there a reason to include a whole new header file just for this? Is it worth it?

Reply:
The type of `size()` for an `std::vector` is `std::size_t`.  The old code works but is the wrong way to do it.
So, yes.